.github/workflows/unittests.yml

```diff
@@ -20,6 +20,7 @@ jobs:
       R_REMOTES_NO_ERRORS_FROM_WARNINGS: true
       R_REMOTES_UPGRADE: never
       VDIFFR_RUN_TESTS: true
+      GITHUB_PAT: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
I don't know. If you look at https://github.com/jasp-stats/jaspAnova/runs/3278537025?check_suite_focus=true you'll see:

![image](https://user-images.githubusercontent.com/21319932/128822348-f39a517a-f82d-47d2-a836-b45eee940b1e.png)

`remotes` fails at `remotes::install_github("jasp-stats/jaspTools")`. I can click the linked URL just fine, so I figured that it might be something authentication-related. If you look at a run with the change you'll see:

![image](https://user-images.githubusercontent.com/21319932/128822567-b05d564a-988f-4135-b09a-47d35f1e35f3.png)

Note that `${{ secrets.GITHUB_TOKEN }}` is btw specific for each GitHub actions run and repository, so if you encounter a similar situation then I think this would be a safe solution.

Reply:
Actually, seeing that Koen had the same issue (https://github.com/jasp-stats/jaspAudit/pull/125#issuecomment-894630374) it's quite likely that every module needs this. I'll try to look into a global solution first though.